### DISCUSSION TITLE
Intellij 重新啟動時若啟動多個專案，再開啟付費功能情況，會抓不到授權

### DISCUSSION THREAD

**Discussion by Sol-LZX** [izhangzhihao/intellij-rainbow-brackets#2631] [Q&A] [answered]
在開啟付費功能時 ( Use color generator for rainbow variables)  只要重啟Intellij 且同時啟動多個專案，授權會無法正常抓取，導致一直無法使用，必須要改用啟動單一專案才可以正常抓到授權

```
IntelliJ IDEA 2023.1 (Ultimate Edition)
Build #IU-231.8109.175, built on March 28, 2023
```

**Comment by Sol-LZX** [comment]
![擷取](https://user-images.githubusercontent.com/47168477/233881018-a6ba5cf7-6211-49b0-85a0-79868ed4847e.JPG)

**Comment by izhangzhihao** [comment]
有可能因为同时启动多个项目时系统资源不够，在license相关组件未初始化好的时候就开始检查license，可能会出现这种情况。你可以尝试 增加IDE的资源，例如内存 来解决问题。

**Comment by izhangzhihao** [comment]
hi，请问问题解决了吗？

**Comment by izhangzhihao** [comment, marked as answer]
Hi, please follow this to check your plugin license status: https://github.com/izhangzhihao/intellij-rainbow-brackets/discussions/2630

**Comment by izhangzhihao** [comment]
I will convert this issue to a discussion, please follow and comment there.

**Comment by Sol-LZX** [comment]
今天2023/05/08我重新測試一次，正常了。

不過我沒記得我有更改什麼， 目前版本為2023.2.7
